### ISSUE TITLE
[Bug] [Mverse] Slight desync is created when the game is paused

### ISSUE THREAD

**Issue by grabz-dev** [KnuckleCracker/CW4-bug-tracker#934]
There is a chance for slight update count desync to be created between players when pausing, the more frequent the pausing the higher likelihood of a big desync being created although it doesn't appear to ever get worse than ~0.4s difference.

How to reproduce: Load up an empty map in mverse with a secodn game on LAN, unpause one game, then rapidly pause the other game. The games' update timers will be out of sync. 

I don't know if this has any bearing on actually desyncing an mverse match or not, if it doesn't, this can be closed.

![image](https://user-images.githubusercontent.com/51026936/123310824-cc6b7500-d526-11eb-84c1-310f036d9035.png)

**Comment by Karsten75** [comment]
Did you happen to make a note oft the latency between the host and client? I'd imagine it's kind of hard to have either react within the bounds of the latency delay introduced by the network and network layer.

**Comment by grabz-dev** [comment]
I was using LAN with both games running on my computer so no real latency to speak of. You should be able to replicate this locally.

**Comment by knucracker** [comment]
It's operating 'normally'.  When I started the whole multiplayer experiment it began as 100% desynchronized.  The only thing I did was share damage on one client to others.  People could be in different maps and different times.  After that, I began a long march towards making things more synchronized.  For the game time, I keep it within 0.5 seconds of the other clients. This is outside the range of most latency situations but still 'close enough' for most player situations.  The greater the flexibility in the time window, the smoother the game will run for players where there is a hickup in time or performance.  That's why I don't try to sync frame by frame.  To help with pausing, I do send a message indicating a game is paused, but a slight latency will build up (up to the max allowed time difference).  Note that even when playing locally,  there will be a small latency.
